Sources/BraintreeThreeDSecure/BTThreeDSecureRequest.swift

```diff
@@ -93,6 +93,10 @@ import BraintreeCore
     /// When using `BTThreeDSecureUIType.native`, all `BTThreeDSecureRenderType` options except `.html` must be set.
     public var renderTypes: [BTThreeDSecureRenderType]?
 
+    /// Optional. Three DS Requester APP URL Merchant app declaring their URL within the CReq message
+    /// so that the Authentication app can call the Merchant app after out of band authentication has occurred. 
+    public var requestorAppURL: String?
```

Comment:
Does this function like an app switch universal or deep link return? Or is just a web-hook/callback to that URL?

Reply:
It's a bit hard to tell, this docstring is pulled directly from the Cardinal docs and they don't make it super clear. It seems like maybe? Apparently it's part of a new Mastercard requirement for merchants that use `prepareLookup`/`initializeChallenge`.